src/cadenza.js

```diff
@@ -323,6 +324,21 @@ export class CadenzaClient {
     return this.#postRequest('setFilter', { filter });
   }
 
+  /**
+   * Set filter Selection in the currently shown map view.
+   *
+   * @param {WorkbookLayerPath} layerPath - The variable values
```

Comment:
Copy/paste mistake: This param is not for the values.

Proposal:
- Name the param just "layer", because the path is an implementation detail.
- Description: "The data view layer to set the selection in"